pkg/daemon/ceph/osd/device_test.go

```diff
@@ -57,31 +57,31 @@ func TestGetOSDInfo(t *testing.T) {
 	assert.Equal(t, testUUID, config.uuid)
 }
 
-func TestOSDBootstrap(t *testing.T) {
-	clusterName := "mycluster"
-	configDir, _ := ioutil.TempDir("", "")
-	defer os.RemoveAll(configDir)
+// func TestOSDBootstrap(t *testing.T) {
```

Comment:
Commented out test for the function I'm not sure is needed or not.

Reply:
may need it for ceph-volume work if we need to create the bootstrap osd keyring. ceph-volume seems to require it.